include/mruby/string.h

```diff
@@ -409,7 +409,7 @@ MRB_API int mrb_str_cmp(mrb_state *mrb, mrb_value str1, mrb_value str2);
 MRB_API char *mrb_str_to_cstr(mrb_state *mrb, mrb_value str);
 
 mrb_value mrb_str_pool(mrb_state *mrb, mrb_value str);
-mrb_int mrb_str_hash(mrb_state *mrb, mrb_value str);
+khint_t mrb_str_hash(mrb_state *mrb, mrb_value str);
```

Comment:
Ah, this is the reason you have disclosed `khint_t` to global.

But `mrb_int` is big enough for most of the case (because its modulo is used in the hash functions). If you really care, use `uint32_t` instead of `khint_t`.

Reply:
Fixed, changed `mrb_str_hash` return type to `uint32_t`.